src/feature/followUnfollow/followUnfollowAction.js

```diff
@@ -0,0 +1,89 @@
+import axios from 'axios';
+import { toast } from 'react-toastify';
+import {
+  CLEAR_FOLLOW,
+  FOLLOW_AUTHOR_SUCCESS,
+  UNFOLLOW_AUTHOR_SUCCESS,
+  FOLLOW_AUTHOR_FAIL,
+  UNFOLLOW_AUTHOR_FAIL,
+  GET_FOLLOWING_AUTHOR_SUCCESS,
+  GET_FOLLOWING_AUTHOR_FAIL
+} from './followUnfollowTypes';
+import { BACKEND_URL } from '../../app/common/config/appConfig';
+
+
+export const clearFollowing = () => (dispatch) => dispatch({ type: CLEAR_FOLLOW });
+
+export const getFollowing = () => async (dispatch) => {
+  const { token } = localStorage;
+  const axiosConfig = {
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: token
+    }
+  };
```

Comment:
@raymond42 wouldn't be better if such axiosConfiguration is centralized i.e. put in a file and get imported wherever and whenever one needs them?

Reply:
done